### PR TITLE
Split `NetworkGraph` message handling fns into unsigned and signed (+update bindings)

### PR DESCRIPTION
**Pull request by TheBlueMatt**
Val noted in 748 (https://github.com/rust-bitcoin/rust-lightning/pull/748#discussion_r529101471) that exposing the existing functions as public was a bit lazy. I indicated I'd handle it in a follow-up post-0.0.12, but realized that 0.0.12 didn't actually pass bindings checks because the newly-exposed APIs required additional features in the bindings. Thus, I went ahead and did the fixup here as well as updated the auto-generated bindings successfully.

As a bonus, this drops Travis CI.

### CODE REVIEW

File: fuzz/src/router.rs [hunk 1]
```diff
@@ -129,15 +127,17 @@ pub fn do_test<Out: test_logger::Output>(data: &[u8], out: Out) {
 					msgs::DecodeError::ShortRead => panic!("We picked the length..."),
 					msgs::DecodeError::Io(e) => panic!(format!("{}", e)),
 				}
-			}
+			};
+			assert_eq!(reader.position(), $len as u64);
+			msg
```
Comment (jkczyz) [inline fuzz/src/router.rs:132]: Consider moving this to the `Ok` arm of the `match`.

File: lightning/src/routing/network_graph.rs [hunk 2]
```diff
@@ -710,22 +734,32 @@ impl NetworkGraph {
 		}
 	}
 
-	/// For an already known (from announcement) channel, update info about one of the directions of a channel.
+	/// For an already known (from announcement) channel, update info about one of the directions
+	/// of a channel.
```
Comment (jkczyz) [inline lightning/src/routing/network_graph.rs:738]: s/a/the ?

**Review by jkczyz — APPROVED** [review]
Nice clean-up! Looks like a largely mechanical change.

**Review by valentinewallace — APPROVED** [review]
I like the new API!

**Comment by TheBlueMatt** [comment]
Only changed the minor nits from @jkczyz.

**Pull request merged**